src/components/frame/useFrame.ts

```diff
@@ -1,35 +1,58 @@
 import { RefObject, useEffect, useState } from "react";
-import connectToParent from "penpal/lib/connectToParent";
-import connectToChild from "penpal/lib/connectToChild";
+import { connectToParent, connectToChild } from "penpal";
+// eslint-disable-next-line
+// @ts-ignore
+import connectToParentV4 from "penpal-v4/lib/connectToParent";
+// eslint-disable-next-line
+// @ts-ignore
+import connectToChildV4 from "penpal-v4/lib/connectToChild";
 import { FrameActionsHandler, LegacyFrameActions } from "./frame.actions";
 import { HostActionsHandler, LegacyHostActions } from "./host.actions";
 import { inIframe } from "../../utils";
+import { getLogger } from "../../logger";
+
+const { trace } = getLogger("useFrame");
 
 type Status = "DISCONNECTED" | "CONNECTING" | "CONNECTED";
 
 interface UseParentFrameProps {
   dispatch: HostActionsHandler;
 }
+
 export const useParentFrame = function({
   dispatch
 }: UseParentFrameProps): [boolean, { dispatch: FrameActionsHandler }] {
   const [parentFrameConnection, setParentFrameConnection] = useState<any>();
   const [status, setStatus] = useState<Status>("DISCONNECTED");
 
   useEffect(() => {
-    const run = async (): Promise<void> => {
-      setParentFrameConnection(
-        await connectToParent({
-          methods: {
-            dispatch: dispatch
-          }
-        }).promise
-      );
-      setStatus("CONNECTED");
-    };
     if (inIframe() && status === "DISCONNECTED") {
+      const parentV5 = connectToParent({
```

Comment:
I would suggest for the parent connection to use only one version, v5. the reason we have control over Parent (verifier) and we can upgrade them to support v5/v4 before upgrading the renderers.

as long as verifiers, supports v5 and v4 we can opt to one of the version. In the event one need a connection with penpal v4, can use a specific version (maybe would be nice to add in readme). Also if we support v5 in verifiers, there is no good reason to allow renderer to use v4

Reply:
looks like its still working with parent at only v5, i added an v4 example to play with:

![Screenshot 2021-02-26 at 1 39 02 PM](https://user-images.githubusercontent.com/4774314/109260139-91875800-7838-11eb-88f3-f24036220529.png)